.github/workflows/modem__build-host-tests.yml

```diff
@@ -155,23 +155,22 @@ jobs:
             ${{ env.COMP_DIR }}/docs_gcovr
           if-no-files-found: error
 
-#  show_report_data:
-#    name: Publish-Results
-#    if: github.ref == 'refs/heads/master' || github.repository != 'espressif/esp-protocols'
-#    runs-on: ubuntu-22.04
-#    needs: gcovr_analyzer_esp_modem
-#    steps:
-#      - name: Checkout 🛎️
-#        uses: actions/checkout@v3
-#        with:
-#          persist-credentials: false
-#      - name: Download Artifacts
-#        uses: actions/download-artifact@v1
-#        with:
-#          name: docs_gcovr
-#
-#      - name: Deploy generated docs
-#        uses: JamesIves/github-pages-deploy-action@v4
-#        with:
-#          branch: gh-pages
-#          folder: 'docs_gcovr'
+  show_report_data:
+    name: Publish-Results
+    if: github.ref == 'refs/heads/master' || github.repository != 'espressif/esp-protocols'
+    runs-on: ubuntu-22.04
+    needs: host_test_gcov_esp_modem
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Download Artifacts
```

Comment:
Do you think it would be possible to deploy the pages directly in the job above? (that way we could remove the upload/download artifact section and simplify the workflow significantly) What was the reason to run it in a separate job?